src/patient-registration/patient-registration-validation.tsx

```diff
@@ -4,7 +4,7 @@ export const validationSchema = Yup.object({
   givenName: Yup.string().required('Given name is required'),
   familyName: Yup.string().required('Family name is required'),
   gender: Yup.string()
-    .oneOf(['M', 'F', 'O', 'U'], 'Gender is unspecified')
+    .oneOf(['Male', 'Female', 'Other', 'Unspecified'], 'Gender is unspecified')
```

Comment:
This may cause issues with form submission. When recording a patient's gender, we only submit the first letter of the word (e.g. 'M' for male)